framework/src/modules/token/constants.ts

```diff
@@ -38,19 +38,20 @@ export const TOKEN_ID_LENGTH = CHAIN_ID_LENGTH + LOCAL_ID_LENGTH;
 export const LOCAL_ID_LSK = Buffer.alloc(LOCAL_ID_LENGTH, 0);
 export const CHAIN_ID_LSK = Buffer.from([0, 0, 0, 0]);
 export const TOKEN_ID_LSK = Buffer.from([0, 0, 0, 0, 0, 0, 0, 0]);
-export const USER_SUBSTORE_INITIALIZATION_FEE = '5000000';
-export const TOKEN_ID_STORE_INITIALIZATION = TOKEN_ID_LSK;
-
+export const USER_SUBSTORE_INITIALIZATION_FEE = BigInt(5000000);
 export const ESCROW_SUBSTORE_INITIALIZATION_FEE = BigInt(5000000);
 
+export const TOKEN_ID_STORE_INITIALIZATION = TOKEN_ID_LSK;
+
 export const defaultConfig = {
 	minBalances: [
 		{
 			tokenID: Buffer.alloc(TOKEN_ID_LENGTH, 0).toString('hex'),
-			amount: DEFAULT_MIN_REMAINING_BALANCE,
+			amount: String(DEFAULT_MIN_REMAINING_BALANCE),
```

Comment:
To be consistent, i think it's better to use `toString` for the conversion

Reply:
Yes, but casting using `String(value)` is more consistent with how we cast bigints with `BigInt(value)` 😉 

Anyhow, I changed it all to `toString()`